File: Local/Eye/src/main/java/br/com/eye/controller/ControllerCriptografia.java
package br.com.eye.controller;


import br.com.eye.model.Criptografia;
import java.security.NoSuchAlgorithmException;

public class ControllerCriptografia {

    public static String gerarSenhaHash(String senha, int salt) throws NoSuchAlgorithmException {
        
        return Criptografia.gerarSenhaHash(senha, salt);
    }
}

File: Web/Eye/Eye/Controller/ControllerCriptografia.cs
using System;
using System.Security.Cryptography;
using System.Text;

namespace Eye.Controller
{
    public class ControllerCriptografia
    {
        public static string GerarSenhaHash(string senha, int salt)
        {
            MD5 md5Hash = MD5.Create();
            byte[] data = md5Hash.ComputeHash(Encoding.UTF8.GetBytes($"{senha}{salt}"));

            StringBuilder sBuilder = new StringBuilder();
            for (int i = 0; i < data.Length; i++)
            {
                sBuilder.Append(data[i].ToString("x2"));
            }
            return sBuilder.ToString();
        }

        public static int GerarSalt()
        {
            return new Random().Next();
        }
    }
}